core/migrations/3_deploy_timer.js

```diff
@@ -0,0 +1,8 @@
+const Timer = artifacts.require("Timer");
```

Comment:
Putting this here because I can't comment on a moved file.

Why move support identifiers from step 10 to step 20 rather than preserve the ordering? The only reason I suggest that we keep the ordering is that they were previously separated into DVM-related migrations (1-10), TD-related migrations (11-16), and EMP-related migrations (17-19). This was to make selective deployments a little easier, specifically, deploying the DVM without any contract creators.

Reply:
Ah this was a fat finger on my part. I needed to move the Timer migration earlier because its an upstream dependency for a lot of contracts, but accidentally bumped support identifiers to last. Will fix.

Reply:
Edited
![Screen Shot 2020-04-14 at 18 35 41](https://user-images.githubusercontent.com/9457025/79280626-cbe54200-7e7e-11ea-88ba-842126e78fc9.png)
 order: